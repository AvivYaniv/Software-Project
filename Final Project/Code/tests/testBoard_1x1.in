solve tests/b1_1.txt
print_board
hint 1 1
set 1 1 0 
set 1 1 1
edit tests/b1_1.txt
print_board
generate 0 0 
generate 1 0 
undo
generate 0 1 
undo
generate 1 1