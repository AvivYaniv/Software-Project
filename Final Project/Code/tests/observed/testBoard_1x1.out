Sudoku
------
Enter your command:
------
|    |
------
Enter your command:
------
|    |
------
Enter your command:
Hint: set cell to 1
Enter your command:
------
|    |
------
Enter your command:
------
|  1 |
------
Puzzle solved successfully
Enter your command:
------
|    |
------
Enter your command:
------
|    |
------
Enter your command:
------
|    |
------
Enter your command:
------
|    |
------
Enter your command:
Error: no moves to undo
Enter your command:
------
|  1 |
------
Enter your command:
------
|    |
------
Undo 1,1: from 1 to _
Enter your command:
------
|  1 |
------
Enter your command:
Exiting...
